docs/standard/io/Buffers.md

```diff
@@ -0,0 +1,401 @@
+# System.Buffers
+
+- **Writing**
+    - [IBufferWriter\<T\>](#ibufferwritert)
+       - [Gotchas](#gotchas)
+- **Reading**
+    - [ReadOnlySequence\<T\>](#readonlysequencet)
+       - [Accessing data](#accessing-data)
+       - [Gotchas](#gotchas-1)
+    - [SequenceReader\<T\>](#sequencereadert)
+       - [Accessing data](#accessing-data-1)
+       - [Gotchas](#gotchas-2)
+   
+## [IBufferWriter\<T\>](https://docs.microsoft.com/en-us/dotnet/api/system.buffers.ibufferwriter-1?view=netstandard-2.1)
+
+`IBufferWriter<T>` is a contract for synchronous buffered writing. At the lowest level the interface is extremely simple and allows you to get access to a `Memory<T>` or `Span<T>`, write to it and say how many `T` items were written.
```

Comment:
```suggestion
`IBufferWriter<T>` is a contract for synchronous buffered writing. At the lowest level, the interface is very simple and allows you to get access to a <xref:System.Buffers.MemoryManager%601.Memory> or <xref:System.Memory%601.Span>, write to it and say how many `T` items were written.
```

Reply:
Why would I want to do that?

Reply:
@mairaw  simple is banned by the style guide, esp my style guide. See my commits if you want to see the type of changes I'm making.